src/services/slam/slam.ts

```diff
@@ -12,15 +12,6 @@ export interface Slam extends Resource {
    */
   getPosition: () => Promise<GetPositionResponse>;
```

Comment:
ah, can we change this type name actually?

```suggestion
  getPosition: () => Promise<SlamPosition>;
```